### PR TITLE
Small Generate Rework

### PR DESCRIPTION
**Pull request by DerGoogler**
Added properties from Fox's MMM's [Custom Repo Format](https://github.com/Fox2Code/FoxMagiskModuleManager/blob/master/docs/DEVELOPERS.md#custom-repo-format) documentation.

Also improved the get method from `moduleprop` with a safer way that handles `None` values. I don't know it Fox's MMM already handle null ID entries. 

Also let me know for any changing requests! 😄

### CODE REVIEW

File: generate.py [hunk 1]
```diff
@@ -46,7 +54,7 @@
         }
 
         # Append to skeleton
-        meta["modules"].append(module)
+        meta.get("modules").append(module)
```
Comment (Atrate) [inline generate.py:57]: Can you leave this as it was?

File: generate.py [hunk 2]
```diff
@@ -6,10 +6,18 @@
 # Configuration
 REPO_NAME = "Magisk-Modules-Alt-Repo"
 REPO_TITLE = "Magisk Modules Alt Repo"
+REPO_WEBSITE = "https://github.com/Magisk-Modules-Alt-Repo/json#readme"
```
Comment (Atrate) [inline generate.py:9]: `https://github.com/Magisk-Modules-Alt-Repo/` would be a better fit for the URL

**Comment by Atrate** [comment]
Except for the highlighted changes, looks good, have you somehow tested this locally, @DerGoogler ?

**Review by Atrate — CHANGES_REQUESTED** [review]

**Comment by DerGoogler** [comment]
@Atrate, yeah. I have it test locally.

File: generate.py (hunk 1 above)
Comment (DerGoogler) [reply generate.py:57]: Type an non existing object an see what happens

https://www.w3schools.com/python/trypython.asp?filename=demo_ref_dictionary_get

Comment (Atrate) [reply generate.py:57]: No matter whether you use `.get` or a simple index-like syntax (`["modules"]`), an exception gets thrown if the `modules` key does not exist. Why switch to `get` in such a case?

**Pull request closed without merging**